### ISSUE TITLE
Build issue

### ISSUE THREAD

**Issue by sanity** [raszi/java-progressbar#3]
Tried to use this with JitPack (an awesome service which automatically builds and distributes Maven and Gradle libraries) - but ran into a build error: https://jitpack.io/com/github/raszi/java-progressbar/c3fbf54fa7/build.log

```
:compileJava/build/src/main/java/hu/ssh/progressbar/ProgressBar.java:7: error: unmappable character for encoding ASCII
 * @author KARASZI Istv??n (github@spam.raszi.hu)
                       ^
/build/src/main/java/hu/ssh/progressbar/ProgressBar.java:7: error: unmappable character for encoding ASCII
 * @author KARASZI Istv??n (github@spam.raszi.hu)
                        ^
/build/src/main/java/hu/ssh/progressbar/console/ConsoleProgressBar.java:48: error: unmappable character for encoding ASCII
 * @author KARASZI Istv??n (github@spam.raszi.hu)
                       ^
/build/src/main/java/hu/ssh/progressbar/console/ConsoleProgressBar.java:48: error: unmappable character for encoding ASCII
 * @author KARASZI Istv??n (github@spam.raszi.hu)
```

**Comment by raszi** [comment]
Can't you switch the source code to UTF-8?

**Comment by sanity** [comment]
@raszi The creators of JitPack suggested this change to the build file: http://mrhaki.blogspot.co.uk/2012/06/gradle-goodness-set-java-compiler.html

**Comment by raszi** [comment]
Fixed with 3233be83d992d00b4f1c4098c4b744ed40d33fb0
